### PR TITLE
MinIO UpdateLegal Konvoy

### PR DESCRIPTION
**Pull request by mmcelroy91**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-89881?filter=-1

## Description of changes being made
Placing MinIO language on download pages for Konvoy.  Also removing the purple NOTE version of the legal.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4485.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/1.8/networking/use-istio/index.md
Comment (cmastr) [inline pages/dkp/konvoy/1.8/networking/use-istio/index.md:2]: @mmcelroy91 did we delete this file on purpose or accident? @mackdubya

**Review by cmastr — APPROVED** [review]

**Comment by mmcelroy91** [comment]
Waiting for tar unbundle un-nest verbiage before merge for Konvoy 2.1 and 2.2 secitons.

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**
